test/like.test.js

```diff
@@ -0,0 +1,312 @@
+        expect(res.body.message).to.deep.equal('Successfully claped');
+        expect(res.body.data).to.have.keys('ArticleSlug', 'status', 'claps');
+        expect(res.body.data.claps).to.deep.equal(2);
+        expect(res.body.data.status).to.deep.equal('like');
+        done();
+      });
+  });
+
+  it('should unlike Article', (done) => {
+    chai
+      .request(server)
+      .put('/api/v1/likes/unlike/fakeslug')
+      .set('Authorization', usertoken)
+      .end((error, res) => {
+        if (error) done(error);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message', 'status', 'data');
+        expect(res.body.message).to.deep.equal('Successfully unliked');
+        expect(res.body.data).to.have.keys('ArticleSlug', 'status', 'claps');
+        expect(res.body.data.claps).to.deep.equal(0);
+        expect(res.body.data.status).to.deep.equal('neutral');
+        done();
+      });
+  });
+
+  it('should dislike Article', (done) => {
+    chai
+      .request(server)
+      .put('/api/v1/likes/dislike/fakeslug')
+      .set('Authorization', usertoken)
+      .end((error, res) => {
+        if (error) done(error);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message', 'status', 'data');
+        expect(res.body.message).to.deep.equal('Successfully disliked');
+        expect(res.body.data).to.have.keys('ArticleSlug', 'status', 'claps');
+        expect(res.body.data.claps).to.deep.equal(0);
+        expect(res.body.data.status).to.deep.equal('dislike');
+        done();
+      });
+  });
+  it('should dislike Article', (done) => {
+    chai
+      .request(server)
+      .put('/api/v1/likes/dislike/fakeslug')
+      .set('Authorization', mikeToken)
+      .end((error, res) => {
+        if (error) done(error);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message', 'status', 'data');
+        expect(res.body.message).to.deep.equal('Successfully disliked');
+        expect(res.body.data).to.have.keys('ArticleSlug', 'status', 'claps');
+        expect(res.body.data.claps).to.deep.equal(0);
+        expect(res.body.data.status).to.deep.equal('dislike');
+        done();
+      });
+  });
+  it('should return all Dislike to a certain article ', (done) => {
+    chai
+      .request(server)
+      .get('/api/v1/likes/dislikes/fakeslug')
+      .set('Authorization', usertoken)
+      .end((error, res) => {
+        if (error) done(error);
+        expect(res).have.status(200);
+        expect(res).to.be.an('object');
+        expect(res.body).to.have.keys('message', 'data', 'status');
+        expect(res.body.message).to.deep.equal('Successfully dislike retrieved');
+        expect(res.body.data).to.have.keys('dislikes');
+        done();
+      });
+  });
+  it('should Unlike Article due to dislike secomd time ', (done) => {
```

Comment:
Consider giving a clear description to tests

Reply:
> Consider giving a clear description to tests

worked on it